### PR TITLE
BUG: Fix for Sphinx 3.4+ search

### LINKED ISSUE #220: Search doesn't work
See:

http://ryan-roemer.github.io/sphinx-bootstrap-theme/README.html

### PR DESCRIPTION
**Pull request by larsoner**
Our version of https://github.com/readthedocs/sphinx_rtd_theme/pull/1021 to fix issues due to https://github.com/sphinx-doc/sphinx/issues/8623, i.e., `search.html` not actually searching due to `Uncaught ReferenceError: Stemmer is not defined` in Sphinx 3.4+.

Closes #214 
Closes #220

### CODE REVIEW

File: sphinx_bootstrap_theme/bootstrap/search.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
 {% set title = _('Search') %}
 {% block extrahead %}
   <script type="text/javascript" src="{{ pathto('_static/searchtools.js', 1) }} "></script>
+  <script type="text/javascript" src="{{ pathto('_static/language_data.js', 1) }} "></script>
```
[the remark this answers is not in the document]
Comment (ivoire) [reply sphinx_bootstrap_theme/bootstrap/search.html:14]: Better to drop it. See https://github.com/ryan-roemer/sphinx-bootstrap-theme/pull/214

**Comment by simonschmeisser** [comment]
friendly ping @larsoner @ivoire  would be nice to have a new release with this fix included

**Comment by larsoner** [comment]
CI errors appear to be of the form `Error: Version 3.6 with arch x64 not found` which perhaps makes sense since they are EOL. No idea why the 3.7+ variants got cancelled, though

**Comment by ivoire** [comment]
@simonschmeisser I'm not maintainer of this package so I can't really help you. I'm also hoping for a patch to be merged and a new release to be made.

**Comment by ryan-roemer** [comment]
@larsoner -- CI should now be fixed if you want to catch this PR up to current `master`. 🤞

**Comment by larsoner** [comment]
Merged with `upstream/master` and it's all green now!

**Review by ryan-roemer — APPROVED** [review]
🎉

**Comment by simonschmeisser** [comment]
Nice, now we just need a new release :tada:  (Sorry for being a bit pushy :wink:

**Comment by ryan-roemer** [comment]
@simonschmeisser haha, no worries -- I was working on that the other day, but just got slammed at work. I'll try and get it out today / this week. (I'm doing the new test release workflow then doing our new full release).

**Comment by ryan-roemer** [comment]
Released in https://pypi.org/project/sphinx-bootstrap-theme/0.8.1/

And looks like the search now works, e.g. here: https://ryan-roemer.github.io/sphinx-bootstrap-theme/search.html?q=bootstrap

Thanks!

**Pull request merged**